extension/sidebar-pane.js

```diff
@@ -251,21 +251,28 @@ function _renderBrowsersElement(browsers) {
   return browsersEl;
 }
 
-function _renderTerms(terms, classes) {
+function _renderTerms(terms, classes, url) {
   const containerEl = document.createElement("span");
 
   for (const term of terms) {
-    const termEl = _renderTerm(term, classes);
+    const termEl = _renderTerm(term, classes, url);
     containerEl.appendChild(termEl);
   }
 
   return containerEl;
 }
 
-function _renderTerm(text, classes = []) {
-  const termEl = document.createElement("span");
+function _renderTerm(text, classes = [], url) {
```

Comment:
Adding a new parameter without a default value after one that has a default value makes it impossible to omit the before last parameter.
Before this change you could call `_renderTerm("text")`, omitting the classes since it has a default value. Now, after your change, you can't do this anymore.
So either move `url` before `classes` if it's mandatory, or give a default value to `url`:

`_renderTerm(text, classes = [], url =  null)`